data/deprecated outfits.txt

```diff
@@ -121,6 +127,7 @@ outfit "Mass Expansion"
 # Deprecated to give it a less generic name.
 outfit "Reverse Thruster"
 	category "Engines"
+	tags human
```

Comment:
You're going to say it should be "deep sky" as well, aren't you? I missed that. Will update! 

Incidentally, I _knew_ you of everyone here would be the guy to help me catch errors in tags. : )

Just a note, I noticed some things, like Armageddon Cores, are only sold at one outfitter (Lovelace Advanced), but there wasn't any in-game justification for Lovelace manufacturing reactors. I figure it's possible it's just meant as a shortcut to get them to appear only in advanced outfitters in certain places, and doesn't necessarily mean they should be tagged Lovelace. So I've kept manufacturer tags mostly to obvious things. Another thing is not marking the Hauler III as Southbound, even though they're the only ones selling it, as I can't find any indication the Hauler line is necessarily theirs.